### PR TITLE
A little proofread

### LINKED ISSUE #18: Proofread the rules. Second round.
First round: #16

Proofread the rules in [README.md](https://github.com/morganey-lang/community/blob/aee339850ee8335e2ed83a0ac0397e7a217ba32b/README.md) and submit the fixes and suggestions as a PR when you're ready.
- Assigned to @Newlifer 
- PR reviewers @rexim @Minoru

### PR DESCRIPTION
**Pull request by Newlifer**
Fix #18

### CODE REVIEW

**Review by ForNeVeR — APPROVED** [review]

**Comment by rexim** [comment]
@Newlifer is this issue related to #18 ?

**Comment by rexim** [comment]
Ah, ok I see now, sorry.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ leaders see [Projects.md](Projects.md).
 
 #### Rights ####
 
-- PLs have full admin access to the repos of the projects they lead.
+- PLs have admin access to the repos of the projects they lead.
```
Comment (rexim) [inline README.md:28]: What's wrong with having full admin access to a specific repo?

File: README.md [hunk 2]
```diff
@@ -40,8 +40,8 @@ Owner is a person who participates in administrative activities of the
 
 #### Rights ####
 
-- Owners have full owner access to [@morganey-lang].
-- Owners can use any technical measures available to them to regulate
+- Owners have owner access to [@morganey-lang].
```
Comment (rexim) [inline README.md:43]: What's wrong with having full owner access to an organization?

**Review by rexim — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ForNeVeR) [reply README.md:43]: Re: both. Is there full and non-full admin/owner access in GitHub model of security?

Comment (rexim) [reply README.md:43]: > Is there full and non-full admin/owner access in GitHub model of security?

Yes

Comment (ForNeVeR) [reply README.md:43]: Ok then. I think it's better to put these words back.

Comment (rexim) [reply README.md:43]: Ah, I misunderstood your question, sorry. No, there is no such thing in GitHub security model. And now I see your point. I am gonna reiterate on that PR later. Thanks!

**Comment by Minoru** [comment]
LGTM.

**Review by rexim — APPROVED** [review]

**Comment by rexim** [comment]
@ForNeVeR @Minoru thanks for the review!
@Newlifer thanks for the proofreading!

**Pull request merged**